gengetopt.yaml

```diff
@@ -90,7 +90,7 @@ test:
             option "dependant" D "Depends on str-opt" int optional dependon="str-opt"
         ' > test.ggo
         gengetopt --input=test.ggo
-        ls -l | grep cmdline.c
+        ls -l cmdline.c 2>/dev/null || echo "cmdline.c not found"
```

Comment:
this _was_ going to fail if there was no 'cmdline.c' file.  (because grep would have not found 'cmdline.c' in 'ls -l' output and woudl have exited non-zero).

Now you're guaranteed to swallow that error with a 'cmdline.c not found' message that no one will ever read. I don't think that is what was intended.

if we just want to see if 'cmdline.c' exists, 
`[ -f cmdline.c ]` or more helpfully:
`[ -f "cmdline.c" ] || { echo "FATAL: cmdline.c did not exist"; exit 1; }'`